### DISCUSSION TITLE
How can graphic move the second rectangle

### DISCUSSION THREAD

**Discussion by pcbren1** [pixijs/pixijs#7577] [Q&A]
How can graphic move the second rectangle
(in order to pursue efficiency, use one graphic instead of instantiating multiple graphics.) 
![99](https://user-images.githubusercontent.com/86771584/124341820-c98d1600-dbf1-11eb-8a46-ec541fe9ce67.png)

**Comment by eXponenta** [comment]
There are not way, because in GPU this is single object.
Rebuilding (this require for moving) will more costly than spawning a 2 graphics.

If you need a billions quads/circles - you can use a mesh. 

https://www.pixiplayground.com/#/edit/omLYwx5VkDCKMsRfNqWuJ
